src/test/java/uk/ac/ebi/eva/pipeline/jobs/steps/VepInputGeneratorStepTest.java

```diff
@@ -93,5 +109,6 @@ public void shouldGenerateVepInput() throws Exception {
 
         assertTrue(vepInput.exists());
         assertEquals("20\t60343\t60343\tG/A\t+", readFirstLine(vepInput));
+        assertEquals(EXPECTED_NON_ANNOTATED_VARIANTS, getLines(new FileInputStream(vepInput)));
     }
```

Comment:
I mean, there's a tricky issue, where we require a non-empty studyId in the genotyped-vcf-job, so only one study will be annotated, but for the annotation-job, we allow both a studyId or an empty string for all studies, and a different stepParametersValidator will need to be done